toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -126,6 +161,19 @@ type LiveOSIsoBuilder struct {
 	cleanupDirs []string
 }
 
+// getArchitecture
+//
+//	Returns the architecture of the current machine.
+//	Supported architectures are amd64 and arm64.
+func getArchitecture() (string, error) {
```

Comment:
A lot of the code that uses `getArchitecture` wants a copy of the `BootFilesArchConfig` for the arch. So, it might be worth adding this as an additional return value.